stdpopsim/generic_models.py

```diff
@@ -1,70 +1,34 @@
 # generic_models
 import msprime
-import stdpopsim.models as models
 
 
 #########
-# Generic Demographic Models
+# Generic Model Mixin class
+# provides generic models that can be
+# initialized with methods
 #
 #
 ########
 
-class GenericConstantSize(models.Model):
-    """
-    Model Name:
-        GenericConstantSize
 
-    Model Description:
-        single-size, single-population model
-
-    Model population indexes:
-        - Population1 : 0
-
-    CLI help:
-        python -m stdpopsim homo-sapiens generic_constant_size -h
-
-    Citation:
-
-    """  # noqa: E501
-
-    def __init__(self, N=1):
+class GenericMixin(object):
```

Comment:
Might be a bit more OO to have 
```
class ConstantSizeMixin(object):
    # Standard docs so that the CLI etc will pick these up and we don't have to duplicate them
    def __init__(self, N):
          self.population_configurations = [
            msprime.PopulationConfiguration(initial_size=N)
        ]
        self.migration_matrix = [[0]]
        self.demographic_events = []

class TwoEpochMixin(object):
     # as before
```

Reply:
can do